### PR TITLE
Session in input

### PR DESCRIPTION
**Pull request by kenneth-miura**
Now u put in a string like "TST101F" and Controller parses out what session you want. Very cool

### CODE REVIEW

**Comment by kenneth-miura** [comment]
Now course input is expected to be in format CSC207F basically. I also made it so the course input will re-prompt user if they put in a course input that doesn't match the format.

**Review by BakerWJ — APPROVED** [review]
lgtm - u can remove the main method if u want

File: src/main/java/workers/ScheduleExporter.java [hunk 1]
```diff
@@ -184,4 +187,15 @@ private static LocalDate getStartingWeekDate(LocalDate termStart, DayOfWeek dow)
             return res;
         }
     }
+
```
Comment (roryhofland) [inline src/main/java/workers/ScheduleExporter.java:190]: IDK if this should be a main method, not high priority fix though

**Review by EclipseIsDead — APPROVED** [review]
Main method removal and we good

**Review by roryhofland — APPROVED** [review]

**Pull request merged**